packages/keystone/prepareKeystoneApp.js

```diff
@@ -18,6 +21,19 @@ async function prepareKeystoneExpressApp (entryPoint, { excludeApps } = {}) {
     // not a csrf case: used for test & development scripts purposes
     // nosemgrep: javascript.express.security.audit.express-check-csurf-middleware-usage.express-check-csurf-middleware-usage
     const app = express()
+
+    const knownProxies = JSON.parse(conf['TRUSTED_PROXIES_CONFIG'] || '{}')
+    // NOTE: from https://expressjs.com/en/guide/overriding-express-api.html
+    Object.defineProperty(app.request, 'ip', {
```

Comment:
What about all other express apps?

Reply:
Currently we're proxying only to KS-based apps